### ISSUE TITLE
test-net doesn't start.

### ISSUE THREAD

**Issue by trustfarm-dev** [digibyte/digibyte#116]
testnet doesn't start.

build latest sourcecode.

digibyte-testnet.conf
```
testnet=1
rpcuser=sssss
rpcpassword=ssssssssssssssssss
rpcport=14023
port=12025
daemon=1
listen=1
server=1
algo=xxxxxx

```

Can't go progress of testnet block sync.
logs

```
2018-03-16 09:27:25 DigiByte Core version v6.16.1.0-ad87d31-DGB (release build)
2018-03-16 09:27:25 InitParameterInteraction: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1
2018-03-16 09:27:25 Assuming ancestors of block 308ea0711d5763be2995670dd9ca9872753561285a84da1d58be58acaa822252 have valid signatures.
2018-03-16 09:27:25 Setting nMinimumChainWork=00000000000000000000000000000000000000000000001f057509eba81aed91
2018-03-16 09:27:25 Using the 'sse4' SHA256 implementation
2018-03-16 09:27:25 Using RdRand as an additional entropy source
2018-03-16 09:27:25 Default data directory /home/nomp/.digibyte
2018-03-16 09:27:25 Using data directory /home/nomp/nomp_chaindata/digibytecoin-test/testnet3
2018-03-16 09:27:25 Using config file /home/nomp/nomp_chaindata/digibytecoin-test/digibyte.conf
2018-03-16 09:27:25 Using at most 125 automatic connections (1024 file descriptors available)
2018-03-16 09:27:25 Using 16 MiB out of 32/2 requested for signature cache, able to store 524288 elements
2018-03-16 09:27:25 Using 16 MiB out of 32/2 requested for script execution cache, able to store 524288 elements
2018-03-16 09:27:25 Using 2 threads for script verification
2018-03-16 09:27:25 scheduler thread start
2018-03-16 09:27:25 HTTP: creating work queue of depth 16
2018-03-16 09:27:25 Config options rpcuser and rpcpassword will soon be deprecated. Locally-run instances may remove rpcuser to use cookie-based auth, or may be replaced with rpcauth. Please see share/rpcuser for rpcauth auth generation.
2018-03-16 09:27:25 HTTP: starting 4 worker threads
2018-03-16 09:27:25 Using wallet directory /home/nomp/nomp_chaindata/digibytecoin-test/testnet3/wallets
2018-03-16 09:27:25 init message: Verifying wallet(s)...
2018-03-16 09:27:25 Using BerkeleyDB version Berkeley DB 4.8.30: (April  9, 2010)
2018-03-16 09:27:25 Using wallet wallet.dat
2018-03-16 09:27:25 CDBEnv::Open: LogDir=/home/nomp/nomp_chaindata/digibytecoin-test/testnet3/wallets/database ErrorFile=/home/nomp/nomp_chaindata/digibytecoin-test/testnet3/wallets/db.log
2018-03-16 09:27:25 Selected Algo: groestl
2018-03-16 09:27:25 Cache configuration:
2018-03-16 09:27:25 * Using 2.0MiB for block index database
2018-03-16 09:27:25 * Using 8.0MiB for chain state database
2018-03-16 09:27:25 * Using 440.0MiB for in-memory UTXO set (plus up to 286.1MiB of unused mempool space)
2018-03-16 09:27:25 init message: Loading block index...
2018-03-16 09:27:25 Opening LevelDB in /home/nomp/nomp_chaindata/digibytecoin-test/testnet3/blocks/index
2018-03-16 09:27:29 Opened LevelDB successfully
2018-03-16 09:27:29 Using obfuscation key for /home/nomp/nomp_chaindata/digibytecoin-test/testnet3/blocks/index: 0000000000000000
2018-03-16 09:27:29 LoadBlockIndexDB: last block file = 0
2018-03-16 09:27:29 LoadBlockIndexDB: last block file info: CBlockFileInfo(blocks=1, size=228, heights=0...0, time=2018-01-26...2018-01-26)
2018-03-16 09:27:29 Checking all blk files are present...
2018-03-16 09:27:29 LoadBlockIndexDB: transaction index disabled
2018-03-16 09:27:29 Opening LevelDB in /home/nomp/nomp_chaindata/digibytecoin-test/testnet3/chainstate
2018-03-16 09:27:33 Opened LevelDB successfully
2018-03-16 09:27:33 Using obfuscation key for /home/nomp/nomp_chaindata/digibytecoin-test/testnet3/chainstate: f5b285ed7478c9b3
2018-03-16 09:27:33 Loaded best chain: hashBestChain=308ea0711d5763be2995670dd9ca9872753561285a84da1d58be58acaa822252 height=0 date=2018-01-26 04:04:34 progress=0.000000
2018-03-16 09:27:33 init message: Rewinding blocks...
2018-03-16 09:27:34 init message: Verifying blocks...
2018-03-16 09:27:34  block index            8772ms
2018-03-16 09:27:34 init message: Loading wallet...
2018-03-16 09:27:34 nFileVersion = 6160100
2018-03-16 09:27:34 Keys: 2001 plaintext, 0 encrypted, 2001 w/ metadata, 2001 total
2018-03-16 09:27:36  wallet                 1292ms
2018-03-16 09:27:36 setKeyPool.size() = 2000
2018-03-16 09:27:36 mapWallet.size() = 0
2018-03-16 09:27:36 mapAddressBook.size() = 0
2018-03-16 09:27:36 mapBlockIndex.size() = 1
2018-03-16 09:27:36 nBestHeight = 0
2018-03-16 09:27:36 AddLocal(175.x.x.x:12025,1)
2018-03-16 09:27:36 Discover: IPv4 enp3s0: 175.x.x.x
2018-03-16 09:27:36 Imported mempool transactions from disk: 0 succeeded, 0 failed, 0 expired, 0 already there
2018-03-16 09:27:36 torcontrol thread start
2018-03-16 09:27:36 Bound to [::]:12025
2018-03-16 09:27:36 Bound to 0.0.0.0:12025
2018-03-16 09:27:36 init message: Loading P2P addresses...
2018-03-16 09:27:36 Loaded 0 addresses from peers.dat  0ms
2018-03-16 09:27:36 init message: Loading banlist...
2018-03-16 09:27:36 init message: Starting network threads...
2018-03-16 09:27:36 net thread start
2018-03-16 09:27:36 dnsseed thread start
2018-03-16 09:27:36 Loading addresses from DNS seeds (could take a while)
2018-03-16 09:27:36 opencon thread start
2018-03-16 09:27:36 init message: Done loading
2018-03-16 09:27:36 addcon thread start
2018-03-16 09:27:36 msghand thread start
2018-03-16 09:27:36 0 addresses found from DNS seeds
2018-03-16 09:27:36 dnsseed thread exit
2018-03-16 09:28:37 Adding fixed seed nodes as DNS doesn't seem to be available.
2018-03-16 09:38:40 Potential stale tip detected, will try using extra outbound peer (last tip update: 630 seconds ago)
2018-03-16 09:49:11 Potential stale tip detected, will try using extra outbound peer (last tip update: 1261 seconds ago)
2018-03-16 09:59:41 Potential stale tip detected, will try using extra outbound peer (last tip update: 1891 seconds ago)
2018-03-16 10:10:12 Potential stale tip detected, will try using extra outbound peer (last tip update: 2522 seconds ago)
2018-03

```

**Comment by i5hi** [comment]
Facing the exact same issue. 
Please help.

**Comment by ChillingSilence** [comment]
Resolved in upcoming 6.17.1
